lib/ZendCode/VarTag.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @package ZendCode
+ */
+namespace Wsdl2PhpGenerator\ZendCode;
+
+use Zend\Code\Generator\DocBlock\Tag\ParamTag;
+
+class VarTag extends ParamTag
```

Comment:
Have you considered submitting this as a pull request to `zendframework/zend-code` instead of having it as a custom class here?